tests/test_audit_basic_update.py

```diff
@@ -69,4 +69,4 @@ def test_audit_basic_update_sets_default_for_contest_is_targeted(client, electio
     assert json.loads(rv.data)["status"] == "ok"
 
     rv = client.get(f"/election/{election_id}/audit/status")
-    assert json.loads(rv.data)["contests"][0]["isTargeted"] == True
+    assert json.loads(rv.data)["contests"][0]["isTargeted"]
```

Comment:
The original assertion was attempting to check if the value was exactly `True`, not truthy. (Though based on the semantics of `==` it might not have been doing that)

Reply:
Alright, that seems worth preserving. I added comments to disable the rules in these tests.

Reply:
Could you change the test to use `is True`?